config/Docker/sanitizer_tests.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
-tests=(
-    tests/helics/core/core-tests
-    tests/helics/common/common-tests
-    "tests/helics/system_tests/system-tests --gtest_filter=-*realtime*"
-	build/tests/helics/apps/helics_apps-tests
-)
-
-SUMRESULT=0
-for test in "${tests[@]}"; do
-    echo "${test}"
-    eval "${test}"
-    RESULT=$?
-    echo "***Latest test result: "${RESULT}
-    SUMRESULT=$(( SUMRESULT + RESULT ))
-done
-# Return 0 or a positive integer for failure
-exit ${SUMRESULT}
-
```

Comment:
Is there a similar file to this one in the scripts folder.  I think at some point I was planning on shifting the circle ci builds to use a script like this, not sure if there was a duplicate in the scripts folder or not.  If not I think I want a copy of this in that folder so it can be used on the HELICS build tests.

Reply:
I thought something weird was going on with git because this showed up as deleted instead of moved. There is an identical copy of this file in the scripts/ folder: https://github.com/GMLC-TDC/HELICS/blob/move-dockerfiles/scripts/sanitizer_tests.sh